### ISSUE TITLE
"Single column" button text gets ellipsed

### ISSUE THREAD

**Issue by pernyl** [qlik-oss/sn-list-objects#140]
The text "Single column" gets ellipsed with ... when a scroll appears in the side-panel (at least on windows). See attached video below.

A workaround could be to style the width of the scrollbar to leave some more space for the content in the side-panel.  

1: Open sub-panel
2: open presentation
3: Click grid button

Scroll bar is seen and button is not looking good

![abc!](https://user-images.githubusercontent.com/29114032/225110858-52a48782-4c3d-4858-8c11-6d162f779fa8.gif)

**Comment by pernyl** [comment]
You can also see this behavior for all of those buttons by using /language/qps-ploc/ so that the "test language" is used.

See image:

![Capture](https://user-images.githubusercontent.com/29114032/225899786-4017e93a-b735-4c19-aed1-e5882a95240a.PNG)

**Comment by CostInCode** [comment]
Tried a solution by decreasing the width of the scrollbar. On Chrome it fixes the issue (for English language) if I set the width to 4 px. 
However, on Firefox it is not possible to explicitly set the scrollbar width to a number of pixels - the only value to make it narrower is `thin`, which doesn't make it narrow enough to fix the issue.
Probably a better solution is to leave the scrollbar as it is, and change the padding of the button, as Quan had already suggested.

Chrome (scrollbar width 4px):
<img width="296" alt="chrome" src="https://user-images.githubusercontent.com/13185716/227207584-3ec6f522-3d1c-40f8-a0ca-dd49c00ed62b.png">

Firefox (scrollbar width: thin)
<img width="296" alt="firefox" src="https://user-images.githubusercontent.com/13185716/227207678-14d495f5-8774-4a7c-98b3-240217b6b25d.png">
